scrabble.js

```diff
@@ -1,15 +1,155 @@
+const LETTER_VALUES =
+{
+  "a": { "points":  1, "tiles":  9 },
+  "b": { "points":  3, "tiles":  2 },
+  "c": { "points":  3, "tiles":  2 },
```

Comment:
Nice work tying together point values and tile counts in one data structure - very clever.